<launch>
  <arg name="rvizconfig" default="$(find shipbot_ros)/rviz/arm.rviz" />
  <node name="shipbot_rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
  <node name="arm_control_node" pkg="shipbot_ros" type="arm_control_node" output="screen">
    <param name="urdf" value="$(find shipbot_urdf)/urdf/shipbot.urdf"/>
    <param name="cx" value="0"/>
    <param name="cy" value="0.7"/>
    <param name="cz" value="0.4"/>
    <param name="radius" value="0.1"/>
    <param name="rate" value="100"/>
  </node>
</launch>
